pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -47,15 +47,23 @@ buildDotnetModule rec {
   '';
 
   postPatch = ''
+    # We still need this for the tests, even though we build with NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR
+    # See https://github.com/Nexus-Mods/NexusMods.App/issues/1836
     ln --force --symbolic "${lib.getExe _7zz}" src/ArchiveManagement/NexusMods.FileExtractor/runtimes/linux-x64/native/7zz
 
     # for some reason these tests fail (intermittently?) with a zero timestamp
     touch tests/NexusMods.UI.Tests/WorkspaceSystem/*.verified.png
   '';
 
   makeWrapperArgs = [
-    "--prefix PATH : ${lib.makeBinPath [ desktop-file-utils ]}"
-    "--set APPIMAGE ${placeholder "out"}/bin/${meta.mainProgram}" # Make associating with nxm links work on Linux
+    "--prefix PATH : ${lib.makeBinPath runtimeInputs}"
+    # Make associating with nxm links work on Linux
+    "--set APPIMAGE ${placeholder "out"}/bin/${meta.mainProgram}"
+  ];
+
+  runtimeInputs = [
+    _7zz
+    desktop-file-utils
   ];
 
   runtimeDeps = [
```

Comment:
Replying to https://github.com/NixOS/nixpkgs/pull/331150#issuecomment-2303296520 by @Sewer56:
> I think, per your own comment [Nexus-Mods/NexusMods.App#1866 (comment)](https://github.com/Nexus-Mods/NexusMods.App/issues/1866#issuecomment-2289524858) adding ICU might be recommended here; if it's not transitively available already.

It looks like ICU is made available _at runtime_ on the `LD_LIBRARY_PATH` by buildDotnetModule: https://github.com/NixOS/nixpkgs/blob/8fe18a3d271a7063bfe40dc82b419c41aa802951/pkgs/build-support/dotnet/build-dotnet-module/default.nix#L187-L192

Doesn't look like it's made available at build time though (e.g. via `buildInputs` or `nativeBuildInputs`), but that might be done elsewhere :thinking: